src/gui/QvisPluginWindow.C

```diff
@@ -452,26 +452,52 @@ QvisPluginWindow::UpdateWindow(bool doAll)
     if (doAll || selectedSubject == fileOpenOptions)
     {
         listDatabases->clear();
-        listDatabases->setSortingEnabled(true);
-        listDatabases->sortByColumn(1, Qt::AscendingOrder);
+        listDatabases->setSortingEnabled(false);
 
         databaseItems.clear();
         databaseIndexes.clear();
-        for (int i=0; i<fileOpenOptions->GetNumOpenOptions(); i++)
-        {            
+
+        // Necessary to sort the plugin names so they appear in 
+        // alphabetical case-insensitive order.
+        std::vector<std::pair<std::string, int>> plugins;
+        for (int i = 0; i < fileOpenOptions->GetNumOpenOptions(); i ++)
+        {
+            // Here we populate the list of plugin names.
+            std::string plugin_name = fileOpenOptions->GetTypeNames()[i].c_str();
+            std::transform(
+                plugin_name.begin(), 
+                plugin_name.end(), 
+                plugin_name.begin(),
+                [](unsigned char c) { return std::tolower(c); });
```

Comment:
This and the `std::sort` call below appear to be using lamda expressions. Is that right? I dunno about others but I find this particular usage (which is also demonstrated on [cppreference.com](https://en.cppreference.com/w/cpp/algorithm/sort)) a bit unreadable. I think its clearer if we just create a file-scope static function that we reference here. But, maybe thats because I am not accustomed to reading C++ code with lambdas.

Reply:
@cyrush what are your thoughts on this?

Reply:
> This and the `std::sort` call below appear to be using lamda expressions. Is that right? I dunno about others but I find this particular usage (which is also demonstrated on [cppreference.com](https://en.cppreference.com/w/cpp/algorithm/sort)) a bit unreadable. I think its clearer if we just create a file-scope static function that we reference here. But, maybe thats because I am not accustomed to reading C++ code with lambdas.

Yes these are lambdas. If you think this is not the best practice I am happy to change it up to be more readable and consistent with standards.

Reply:
while they take a bit to get used to, I think lambas are actually better in cases like this. 

Creating and naming a new function adds a different type of mental overhead.

Reply:
> Creating and naming a new function adds a different type of mental overhead

Yes and no...at least that type of *mental overhead* is one anyone who has been programming C/C++ for any length of time is already used to.

All that said, I did approve already. I dunno if my combination of making a remark about what I think would be clearer combined with an approval winds up confusing anyone. If so, lemme know.

My intention in so doing is to mention something maybe worth taking a closer look at if one is interested or so inclined but to not hold up making progress if one is not.